.github/workflows/build.yml

```diff
@@ -0,0 +1,31 @@
+name: Run tests and build package
+
+on: pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Install system dependencies
+      run: |
+        sudo apt update && sudo apt install -y --no-install-recommends python3-setuptools
+        pip3 install --upgrade setuptools
+        pip3 install wheel twine flake8 black
+
+    - name: Build package
+      run: python3 setup.py sdist bdist_wheel
+
+    - name: Twine version
+      run: python3 -m twine --version
+
+    - name: Twine check
+      run: python3 -m twine check dist/*
+
+    - name: Lint Python
+      run: python3 -m flake8 canonicalwebteam tests setup.py && python3 -m black --line-length 79 --check canonicalwebteam tests setup.py
+
+    - name: Test Python
+      run: python3 setup.py test
```

Comment:
We used to have it so that we could see a different check in the PR for each item. So ideally, we should be able to see 3 different lines in the PR for each of:

- package build / check failed
- linter failed
- tests failed

I think the only way to achieve this in Actions is to create 3 different workflows. Would that be too cumbersome? Could we do that?

@tbille, @jpmartinspt, @carkod please add comments if you like.

Reply:
There is a way, I pointed @jkfran to the Flask-Base workflow and he will update.

Reply:
I have divided it into different jobs within the workflow.